libraries/GCS_MAVLink/GCS.h

```diff
@@ -148,7 +148,7 @@ class GCS_MAVLINK
 #if AP_AHRS_NAVEKF_AVAILABLE
     void send_opticalflow(AP_AHRS_NavEKF &ahrs, const OpticalFlow &optflow);
 #endif
-    void send_autopilot_version() const;
+    void send_autopilot_version(uint8_t major_version = 0, uint8_t minor_version = 0, uint8_t patch_version = 0, enum FIRMWARE_RELEASE_TYPE release_type = DEV) const;
```

Comment:
Yep!  But Randy normally disapproves of them.  They are great for temporary library changes, so that vehicles can use the functions the old way until all vehicles catch up.  Then we make them required arguments again.